seed/views/organizations.py

```diff
@@ -840,3 +840,42 @@ def matching_criteria_columns(self, request, pk=None):
         )
 
         return JsonResponse(matching_criteria_column_names)
+
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_member')
+    @detail_route(methods=['GET'])
+    def geocoding_columns(self, request, pk=None):
```

Comment:
Used to ensure geocoding columns do exist for an org's inventory type as well as to list the geocoding columns in the column mappings page within the import file flow.